### ISSUE TITLE
components: manual position adjustments

### ISSUE THREAD

**Issue by Rapsodia86** [r-tmap/tmap#993]
Hi @mtennekes,
I still working on tm_credits and have another question, but I cannot open the last issue that you closed (#986).
So I am reposting that question here:

Is there a way to make additional (manual) adjustments within a cell by e.g. providing +/- pnt to make extra shift? Or maybe in this case it would be to reduce margins?
I would like to have those credits at the same level as the facet panels in the upper row, but I cannot make it happen 🙃 
It is either below or above

tm_shape(NLD_muni) +
  tm_polygons("pop_0_14",
              fill.scale = tm_scale_intervals(values = "pu_gn", style = "kmeans", n = 7),
              fill.legend =  tm_legend(position = tm_pos_out(pos.v= "center"))) +
  tm_facets_wrap(by = "province")+
  tm_credits("TEST CREDITS", position = tm_pos_out(pos.h = "right", pos.v = "top"))

![image](https://github.com/user-attachments/assets/4de187a1-d974-4ebd-a665-01d470657075)

tm_shape(NLD_muni) +
  tm_polygons("pop_0_14",
              fill.scale = tm_scale_intervals(values = "pu_gn", style = "kmeans", n = 7),
              fill.legend =  tm_legend(position = tm_pos_out(pos.v= "center"))) +
  tm_facets_wrap(by = "province")+
  tm_credits("TEST CREDITS", position = tm_pos_out(cell.v="top",pos.h = "right", pos.v = "bottom"))

![image](https://github.com/user-attachments/assets/9cee2008-c7b9-46b5-9df3-5560f1e62765)

**Comment by Rapsodia86** [comment]
Thank you for checking and catching that bug! 
`tmap_design_mode()` looks very helpful! 
Ok, I see in the documentation you shared last time that there is info about numbers for finetuning https://r-tmap.github.io/tmap/reference/tm_pos.html#arg-pos-h-pos-v. I must have missed that one! 

Thanks again!

**Comment by mtennekes** [comment]
You're welcome.

It's working now. What I would do in to align the credits to the panels:

```r
tm_shape(NLD_muni) +
	tm_polygons("pop_0_14",
				fill.scale = tm_scale_intervals(values = "pu_gn", style = "kmeans", n = 7),
				fill.legend = tm_legend(position = tm_pos_out(pos.v= "center"))) +
	tm_facets_wrap(by = "province")+
	tm_credits("TEST CREDITS", position = tm_pos_out(pos.h = "right", pos.v = "top"), padding = c(0,0,1,0))
```

`pos.v` Aligns it at the top, and `padding` (bottom, left, top, right) adds some padding space. Alternatively, you can set both pos.v and pos.h to numerics (between 0 and 1).

If you have any other question please reopen

**Comment by Rapsodia86** [comment]
Just tested it and it works well!
I do have two more questions but cannot reopen the issue!  
1) I wanted to set pos.h as numeric (for fine-tuning) while pos.v leave as "right" and that gave me an error:  
Error: position argument: for pos.h and .v should both be numeric or character values.
So then for pos.v I set 1, and that worked. Would it be possible to mix numeric and character values? 
2) Could padding take negative values? 

Thank you again!
